forward_engineering/ddlProvider.js

```diff
@@ -102,12 +104,13 @@ module.exports = (baseProvider, options, app) => {
 			const deActivatedColumns = columns.filter(column => !column.isActivated).map(({ column }) => column);
 			const partitionsStatement = commentIfDeactivated(partitions, { isActivated: isPartitionActivated });
 
+			const foreignKeysConstraintsStatements = foreignKeyConstraints.map(({statement}) => statement)
 			const compositePkFieldsNamesList = primaryKey.flatMap(compositePK => compositePK?.compositePrimaryKey.map(({name: columnName}) => columnName))
 			const compositePrimaryKeyOutlineConstraint = compositePkFieldsNamesList.length ? `PRIMARY KEY (${compositePkFieldsNamesList.join(', ')}) NOT ENFORCED`: ''
 			const tableStatement = assignTemplates(templates.createTable, {
 				name: tableName,
 				column_definitions: externalTableOptions?.autodetect ? '' : '(\n' + tab(
-					[activatedColumns.join(',\n'), deActivatedColumns.join(',\n'), compositePrimaryKeyOutlineConstraint].filter(Boolean).join('\n'),
+					[wrapStatementWithComma(activatedColumns.join(',\n')), wrapStatementWithComma(deActivatedColumns.join(',\n')), wrapStatementWithComma(compositePrimaryKeyOutlineConstraint), foreignKeysConstraintsStatements.join(',\n')].filter(script => script.length).join('\n'),
```

Comment:
It will add 2 commas, first in `wrapStatementWithComma`, second in `,join(',\n')`